lib/relay.js

```diff
@@ -316,23 +316,19 @@ function responseHandler(filterRules, config, io) {
         }
 
         const status = (response && response.statusCode) || 500;
-        logResponse(logContext, status, response.headers);
+        logResponse(logContext, status, response, config);
         emit({ status, body, headers: response.headers });
       });
     });
   };
 }
 
-function logResponse(logContext, status, headers) {
+function logResponse(logContext, status, response, config = null) {
   logContext.httpStatus = status;
-  logContext.httpHeaders = headers;
-
-  const logMsg = 'sending response back to websocket connection';
-  if (status <= 200) {
-    logger.debug(logContext, logMsg);
-  } else {
-    logger.info(logContext, logMsg);
-  }
+  logContext.httpHeaders = response.headers;
+  logContext.httpBody = config && config.LOG_ENABLE_BODY === 'true' ? response.body : null;
+
+  logger.info(logContext, 'sending response back to websocket connection');
```

Comment:
is this only used in broker client?

Reply:
Yep, the server part (above) just calls the logger directly.